GithubProfileWiki/Networking/NetworkConstants.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NetworkConstants.swift
+//  GithubProfileWiki
+//
+//  Created by ilter on 23.01.2022.
+//
+
+import Foundation
+struct NetworkConstants {
```

Comment:
i think it is better to keep constant's in enum to avoid create instance of structs accidentally